tests/bluetooth/shell/testcase.yaml

```diff
@@ -29,6 +29,13 @@ tests:
     platform_allow:
       - native_posix
     build_only: true
+  bluetooth.shell.path_loss_monitoring:
+    extra_configs:
+      - CONFIG_BT_PATH_LOSS_MONITORING=y
```

Comment:
```suggestion
      - CONFIG_BT_TRANSMIT_POWER_CONTROL=y
      - CONFIG_BT_PATH_LOSS_MONITORING=y
```

Reply:
Did you forget this?

Reply:
What is the reason for this? Do you mean I should combine with the testcase above?

Reply:
i thought we were against `select BT_CTLR_LE_POWER_CONTROL` in favour of `depends on`.

Refer to: https://github.com/zephyrproject-rtos/zephyr/pull/70251#discussion_r1601487145

> In summary, here are some recommended practices for select:
Avoid selecting symbols with prompts or dependencies. Prefer depends on. If depends on causes annoying bloat in configuration files, consider adding a Kconfig default for the most common value.

@tejlmand FYI

Reply:
As discussed [here](https://github.com/zephyrproject-rtos/zephyr/pull/76175#discussion_r1687276841), `depends on` does not have the desired effect - I presume this is why `select` has been used on other Kconfigs in the file

Reply:
> As discussed [here](https://github.com/zephyrproject-rtos/zephyr/pull/76175#discussion_r1687276841), `depends on` does not have the desired effect - I presume this is why `select` has been used on other Kconfigs in the file

"desired effect" has not been a criteria in the "recommended practices", adding new Kconfig default is another recommended practice (does add more Kconfig... but...) and hence will like @tejlmand to provide his comments.